src/main2.js

```diff
@@ -0,0 +1,19 @@
+import "./js/localstorage.js";
```

Comment:
what's the reason for splitting into main and main2?

Reply:
If you run `npm run build` webpack says you should split the code because the generated javascript ends up being to big. It just to solve that warning

Reply:
that is because we are not compressing the output javascript. that would be the simplest solution

Reply:
@yeiniel  So code splitting doesn't do much, right?

Reply:
Code splitting can have a noticeable impact on the app loading but right now the reason for the message is just that we are not compressing the bundle.